src/lexstate.c

```diff
@@ -92,7 +92,7 @@ static const char *RBS_TOKENTYPE_NAMES[] = {
   "tANNOTATION",      /* Annotation */
 };
 
-rbs_token_t NullToken = { .type = NullType, .range = {} };
+rbs_token_t NullToken = { .type = NullType, .range = {0} };
```

Comment:
You'll need to make the same change here:

https://github.com/ruby/rbs/blob/6c08f20a3dfcba8efebd54da1010738c3c0a3ed1/src/parser.c#L3418

```diff
 .constant_pool = {0}, 
```

Reply:
Reusing `NULL_RANGE` didn't work with clang-15, on GitHub Actions... 😢